src/Control/Concurrent/Async/Lifted/Safe.hs

```diff
@@ -327,6 +327,11 @@ concurrently
   => m a -> m b -> m (a, b)
 concurrently = liftBaseOp2_ A.concurrently
 
+concurrently_
+  :: forall m a b. (MonadBaseControl IO m, Forall (Pure m))
+  => m a -> m b -> m ()
+concurrently_ left right = void $ concurrently left right
```

Comment:
Could we change this to `liftBaseOp2_ A.concurrently_` to make it as close as possible to the `async` package?